server/src/bots/telegram/index.ts

```diff
@@ -65,62 +62,88 @@ export function runTelegramBot(
         // Message handler for feature  Countries / Country
         .registerMessageHandler(
             [UserRegExps.CountriesData, UserMessages.CountriesData],
-            countriesResponse
+            withSingleParameterAfterCommand(
+                countriesResponse,
+            ),
         )
         .registerMessageHandler(
             [UserRegExps.AvailableCountries, UserMessages.AvailableCountries],
-            showAvailableCountriesResponse
+            withSingleParameterAfterCommand(
+                showAvailableCountriesResponse,
+            ),
         )
         .registerMessageHandler(
             [UserRegExps.CountryData],
-            showCountryByNameStrategyResponse
+            withSingleParameterAfterCommand(
+                showCountryByNameStrategyResponse,
+            ),
         )
         // Message handler for feature  Advices
         .registerMessageHandler(
             [UserRegExps.Advice, UserMessages.GetAdviceHowToBehave],
-            showAdvicesHowToBehaveResponse
+            showAdvicesHowToBehaveResponse,
         )
         // Message handler for feature  Help
         .registerMessageHandler(
             [UserRegExps.Help, UserMessages.Help],
-            helpInfoResponse
+            helpInfoResponse,
         )
         // Message handler for feature  Assistant
         .registerMessageHandler(
             [UserRegExps.Assistant, UserMessages.Assistant],
-            assistantStrategyResponse
+            withSingleParameterAfterCommand(
+                assistantStrategyResponse,
+            ),
         )
         // Message handler for feature  Subscriptions
         .registerMessageHandler(
             [UserMessages.SubscriptionManager],
-            subscriptionManagerResponse
+            subscriptionManagerResponse,
         )
         .registerMessageHandler(
             [UserMessages.Existing],
-            showExistingSubscriptionsResponse
+            showExistingSubscriptionsResponse,
         )
         .registerMessageHandler(
             [UserRegExps.Subscribe, CustomSubscriptions.SubscribeMeOn],
-            subscribingStrategyResponse
+            withSingleParameterAfterCommand(
+                subscribingStrategyResponse,
+            ),
         )
         .registerMessageHandler(
             [
                 CustomSubscriptions.UnsubscribeMeFrom,
                 UserRegExps.Unsubscribe,
                 UserMessages.Unsubscribe,
             ],
-            unsubscribeStrategyResponse
+            withSingleParameterAfterCommand(
+                unsubscribeStrategyResponse,
+            ),
         )
         .registerMessageHandler(
             [UserRegExps.Trends],
-            withTwoArgumentsAfterCommand(trendsByCountryResponse)
```

Comment:
I was thinking If we could wrap ```withSingleParameterAfterCommand``` with ```withTwoArgumentsAfterCommand```
That way we won't need to write here both of the decorators. just ```withTwoArgumentsAfterCommand```
@danbilokha what do you think?

Reply:
discussed